docs/source/contributing_playbooks.rst

```diff
@@ -0,0 +1,92 @@
+Contributing Playbooks
+----------------------
+
+Addition of new playbooks to the edpm-ansible collection is easy,
+thanks to specialized development tooling and CI pipeline.
+However, certain requirements must be met by any new playbook,
+in order to ensure quality of the collection and a smooth operation
+of software consuming it.
+
+These requirements, outlined in the following document, go beyond
+syntactic and functional correctness of the code.
+
+Where there isn't an explicit requirement, or recommendation,
+provided by this document, the primary `Ansible documentation`_ is considered
+to be the source of truth. When this document and official Ansible docs
+are in substantial conflict, an issue should be raised.
+
+Playbook design principles
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Playbooks tie multiple roles and tasks together into cohesive whole.
+If roles represent subroutines of Ansible world, playbooks are programs and scripts.
+Still intended for a certain purpose, but utilizing diverse functionalities of different
+roles to achieve it.
+
+Playbook must duplicate other playbooks.
+
+If a newly developed playbook requires functionality not already present in existing
+roles, a new role should be submitted in the same PR implementing it.
+
+Playbooks must use following basic layout. Exposing variables, `edpm_override_hosts`,
+`edpm_max_fail_percentage` and `edpm_any_errors_fatal`.
+
+.. code-block:: YAML
+
+    - name: NEWPLAYNAME
+      hosts: "{{ edpm_override_hosts | default('all') }}"
+      strategy: linear
+      become: true
+      any_errors_fatal: "{{ edpm_any_errors_fatal | default(true) }}"
+      max_fail_percentage: "{{ edpm_max_fail_percentage | default(0) }}"
+
+This allows for more granular error handling and playbook application by operators.
+
+Error handling
+++++++++++++++
+
+Errors encountered while executing playbooks should be handled in roles which
+caused them.
+
+Playbook test development
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Playbooks themselves don't require tests. However, all of the roles they invoke do.
+For further information about testing roles, please refer to the role contribution
+section of this guide.
+
+
+Documenting a new playbook
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Playbooks must accurately explain their effects in their documentation, with in-line
+comments serving only as a supplement. In complex cases the documentation should link
+to other material, but only as a source of additional information. Beware the link rot.
+
+Playbook license must be compatible with the primary repo license `Apache v2.0`.
```

Comment:
why would we allow any license other than the repo license? I don't feel like this statement is needed. Perhaps general contributing guidance should say something like:

All content contributed to edpm-ansible must be licensed the same as edpm-ansible (`Apache v2.0`).

Reply:
In general we could just say "use apache" but there are compatible licenses and I recall we had some cases in tripleo when we had them rather than then the primary. I suppose it's not really likely to be a deal breaker for anyone so I'll just remove the sentence.